assets/js/state/sagas/clusters.js

```diff
@@ -1,6 +1,44 @@
-import { put, takeEvery } from 'redux-saga/effects';
-import { CLUSTER_DEREGISTERED, removeCluster } from '@state/clusters';
+import { call, put, select, takeEvery } from 'redux-saga/effects';
+import { post } from '@lib/network';
+
 import { notify } from '@state/actions/notifications';
+import {
+  CLUSTER_DEREGISTERED,
+  removeCluster,
+  updateSelectedChecks,
+  CLUSTER_CHECKS_SELECTED,
+} from '@state/clusters';
+import {
+  setClusterChecksSelectionSavingError,
+  setClusterChecksSelectionSavingSuccess,
+  startSavingClusterChecksSelection,
+  stopSavingClusterChecksSelection,
+} from '@state/clusterChecksSelection';
+import { getClusterName } from '@state/selectors/cluster';
+
+function* checksSelected({ payload }) {
```

Comment:
No test for this? I know that it is a migration, but it looks a good opportunity to add tests

Reply:
My idea was to shortly converge to just one `assets/js/sagas/checksSelection.js` for both clusters and hosts, and basically move there what we have in `assets/js/state/sagas/hosts.js` about `checksSelected` and related tests.

And remove this one